homeassistant/components/blebox/config_flow.py

```diff
@@ -46,6 +53,8 @@ def create_schema(previous_input=None):
         {
             vol.Required(CONF_HOST, default=host): str,
             vol.Required(CONF_PORT, default=port): int,
+            vol.Optional(CONF_USERNAME, default=""): str,
+            vol.Optional(CONF_PASSWORD, default=""): str,
```

Comment:
```suggestion
            vol.Optional(CONF_USERNAME): str,
            vol.Optional(CONF_PASSWORD): str,
```

By removing the default value, the key will not be included in the  dict and we can use `dictionary.get(key):` directly without the helper `get_non_empty_key`.
`.get` returns none if key is not present and no default value is specified.